c2pa/c2pa_api/c2pa_api.py

```diff
@@ -36,9 +36,12 @@
 # reader = Reader("image/jpeg", open("test.jpg", "rb"))
 # json = reader.json()
 class Reader(api.Reader):
-    def __init__(self, format, stream):
+    def __init__(self, format, stream, manifest_data=None):
         super().__init__()
-        self.from_stream(format, C2paStream(stream))
+        if manifest_data is not None:
```

Comment:
Is there any chance that the manifest_data could be blank (e.g. ` ` or any variation of whatever could be considered blank). Do we need special care around that?

Reply:
This option is allow verifying a sidecar or remote manifest. If we have retrieved a .c2pa manifest store from our cloud or a sidecar, we can pass it in along with an associated asset to see if it verifies. If the manifest_data is not a correctly structured binary .c2pa we will return errors reporting it.
I should add a ticket somewhere to do more type checking at a higher level, If you pass the wrong type of parameter, it will get caught, but the error messages are obscure ones from the binding layer.